### DISCUSSION TITLE
[Feature] Import and display Artist and Copyright metadata

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8713] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have many photos in my library that were either taken by other people or scanned and the originals don’t belong to me. Most of them have metadata written by me in the fields - "Artist" and "Copyright".

It would be great if Immich could also store that information and import it from EXIF metadata during upload.

Both information should be present in the details pane of specific photos and on the caption bar when that bar is eventually implemented #8710

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Related: #8365

**Comment by marank** [comment]
This would be great! I have tons of pictures which I got from friends, family and others. I would love to see the Artist tag in the details pane.

**Comment by stefaanvp** [comment]
Me too. I work together with a lot of photographers and we share our work all the time. Immich is great, but I really need to see who took the picture.
